_i18n/fr.yml

```
@@ -89,7 +90,7 @@ home:
     confidentialité et avec des frais minimes, contre des biens ou des services. Nombre
     d'entités  acceptent le Monero (XMR), regardez notre liste de commerçants qui
     acceptent actuellement le Monero.
-  useit: "C'est une monnaie : utilisez-la !"
+  useit: "C'est une monnaie; Utilisez-la !"
```

```suggestion
  useit: "C'est une monnaie; Utilisez-la!"
```
maybe not an issue but i noticed the "!" on a new line because of screen width and at least some other translations of no space between the "!"

---

In French they put a space before question and exclamation marks, but i don't think it's an issue if it's inconsistent here and there. More annoying to have the ! in a newline, but doesn't worth changing structure because of it. Will look what can be done.